src/sentry/api/serializers/models/project.py

```diff
@@ -124,6 +125,9 @@ def get_access_by_project(
 
             # User may have elevated team-roles from their org-role
             top_org_role = org_roles[0] if org_roles else None
+            if is_superuser:
```

Comment:
Not sure how to test this code branch, given that `is_superuser` is checking `env.request` and the serializer tests doesn't use logins.